collectors/mimecast/utils.js

```diff
@@ -57,6 +57,7 @@ function getAPILogs(authDetails, state, accumulator, maxPagesPerInvocation) {
                             if (body.data && body.data.length > 0) {
                                 accumulator.push(...body.data);
                             }
+                            AlLogger.debug(`MIME000011 accumulated first element: ${accumulator[1]}`);
                             if (response.meta && response.meta.isLastToken) {
                                 nextPage = undefined;
                                 if(applicationDetails.payload.data[0].token){
```

Comment:
please remove the line [66](https://github.com/alertlogic/paws-collector/pull/283/files#diff-0e2b97087b6b0490419fc24e481b93be7b5b00573dc973e2597e557fc4a1718eR66).